src/components/html.js

```diff
@@ -123,4 +123,4 @@ function Html({ html, ...rest }) {
   return <Prose {...rest}>{renderHast(tree)}</Prose>;
 }
 
-export default Html;
+export default Html;
```

Comment:
I was going to add a new line for you, but the whole file needs linting fixes. This can be taken care of whenever you get around to removing Unified/Rehype.